organize/models.py

```diff
@@ -104,9 +105,10 @@ def deploy(self):
 
         self.change_status_to(DEPLOYED)
 
-        # TODO: we should recognize here if we should create a new event,
-        # copy old one or copy old and change organizaers.
-        event = self.create_event()
+        if self.previous_event:
```

Comment:
I think we may need to do more than that here... In the form, we ask user whether they've organized an event before, and if so, we put it in `self.previous_event`. That doesn't mean that there wasn't an event in this city before. I think we should use something like `Event.objects.filter(city=city, country=country).exists()` here.